tests/state/conftest.py

```diff
@@ -0,0 +1,312 @@
+import pytest
+import random
+
+from beacon_chain.state.active_state import (
+    ActiveState,
+)
+from beacon_chain.state.aggregate_vote import (
+    AggregateVote,
+)
+from beacon_chain.state.block import (
+    Block,
+)
+from beacon_chain.state.crosslink_record import (
+    CrosslinkRecord,
+)
+from beacon_chain.state.crystallized_state import (
+    CrystallizedState,
+)
+from beacon_chain.state.validator_record import (
+    ValidatorRecord,
+)
+from beacon_chain.state.state_transition import (
+    ATTESTER_COUNT,
+    DEFAULT_BALANCE,
+    DEFAULT_SWITCH_DYNASTY,
+    EPOCH_LENGTH,
+    MAX_VALIDATORS,
+    SHARD_COUNT,
+    compute_state_transition,
+    get_attesters_and_signer,
+    get_crosslink_aggvote_msg,
+    get_shard_attesters,
+    get_shuffling,
+    state_hash,
+)
+
+import beacon_chain.utils.bls as bls
+from beacon_chain.utils.blake import (
+    blake,
+)
+from beacon_chain.utils.simpleserialize import (
+    serialize,
+)
+
+
+DEFAULT_SHUFFLING_SEED = b'\35'*32
+DEFAULT_RANDAO = b'\45'*32
+DEFAULT_NUM_VALIDATORS = 40
+
+
+@pytest.fixture
+def sample_active_state_params():
+    return {
+        'height': 30,
+        'randao': b'\x35'*32,
+        'ffg_voter_bitfield': b'\x42\x60',
+        'balance_deltas': [1, 2, 3],
+        'recent_attesters': [0, 2, 10],
+        'partial_crosslinks': [],
+        'total_skip_count': 33,
+        'recent_proposers': []
+    }
+
+
+@pytest.fixture
+def sample_recent_proposer_record_params():
+    return {
+        'index': 10,
+        'randao_commitment': b'\x43'*32,
+        'balance_delta': 3
+    }
+
+
+@pytest.fixture
+def init_shuffling_seed():
+    return DEFAULT_SHUFFLING_SEED
+
+
+@pytest.fixture
+def init_randao():
+    return DEFAULT_RANDAO
+
+
+@pytest.fixture
+def attester_count():
```

Comment:
hmmm... I actually like to expose config vars like this as their own fixture so I can run tests on the global configuration and just change one param as a fixture. Something like the following:

```python
@pytest.mark.parametrize(
    'epoch_length',
    [(0), (10), (100000)]
)
test_different_epoch_lengths(epoch_length, mock_make_child):
    do_stuff()
```

This will change the base `epoch_length` fixture and ripple through the other fixtures. In this case it will pass `epoch_length` to the `config` fixture, thus passing in various `epoch_length` values to `mock_make_child` via `config`.

I'll create the config.py file and then think about how I want to handle this case.

Reply:
Sounds good. :)